src/creeps/Zombie.cpp

```diff
@@ -1,76 +1,290 @@
 #include <math.h>
 #include <random>
 #include <cassert>
-
-#include "../creeps/Zombie.h"
+#include <utility>
+#include "Zombie.h"
+#include "Node.h"
 #include "../game/GameManager.h"
 #include "../log/log.h"
+using namespace std;
 
 #define PI 3.14159265
```

Comment:
`#define _USE_MATH_DEFINES`
can allow you to use the built in PI
[check here](http://stackoverflow.com/a/1727896)